Source/Extensions/Blazorise.Snackbar/Snackbar.razor.cs

```diff
@@ -44,7 +46,12 @@ public partial class Snackbar : BaseComponent, IDisposable
     private CountdownTimer countdownTimer;
 
     /// <summary>
-    /// Flag that indicates if snackbar close action was delayed.
+    /// A flag that indicates if the snackbar is currently being closed with animation.
+    /// </summary>
+    private bool closeAnimation = false;
```

Comment:
Would `closingAnimation` be a better naming then? It would be consistent with `closingDelayed` and the documentation we've added also talks about "being closed" => closing

Reply:
Yes, definitely